### ISSUE TITLE
Ready-to-go: PR for adding interactivity to lines and polygons, but encountering accessibility-related linter issues

### ISSUE THREAD

**Issue by nickw1** [omnidotdev/rdk#119]
### Problem

Allows `GeoLine` and `GeoPolygon` to be clickable.

### Proposed Solution

`onClick`, `onPointerOut` and `onPointerOver` attributes added to `GeoLine` and `GeoPolygon` and forwarded to the appropriate underlying mesh or group.

This has been done and tested. However the linter (on git commit) is failing at the accessibility check, commenting that the elements do not have an ARIA `role`. Looking at the docs https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles the most appropriate role seems to be `button` (for a clickable element), however if I add a `button` role, however if I do that, the linter is asking me to change the R3F component to a `button`.

I guess this is a linter config issue, @coopbri presumably this needs an alteration to the linter rules? Once it's handled, there's a working PR for this issue tested and all ready to go.

**Comment by nickw1** [comment]
Have read into Biome configuration and turned off the appropriate accessibility setting as a quick-and-dirty way of getting the commit through. Not sure if this is acceptable though, but will open the PR and mention it there.
